libs/shared/ui/src/lib/components/chart/chart.stories.tsx

```diff
@@ -252,14 +252,24 @@ The chart supports mixed visualization types including area charts, bar charts,
           />
           <Tooltip
             content={
-              <Chart.TooltipContent
-                title="System Usage"
-                formatLabel={(key) => {
-                  const labelMap: Record<string, string> = { cpu: 'CPU', memory: 'Memory', disk: 'Disk' }
-                  return labelMap[key] || key
-                }}
-                formatValue={(value) => `${value}%`}
-              />
+              zoomState.refAreaLeft && zoomState.refAreaRight ? (
+                <Chart.TooltipContent
```

Comment:
Do you think we can have an custom component tooltip for it ? To keep the TooltipContent more generic and avoid having too much props in this component?